deepmd/utils/argcheck.py

```diff
@@ -1493,6 +1493,10 @@ def dpa3_repflow_args():
         "Whether to enable the optimized update method. "
         "Uses a more efficient process when enabled. Defaults to True"
     )
+    doc_smooth_edge_update = (
+        "Whether to make edge update smooth."
```

Comment:
```suggestion
        "Whether to make edge update smooth. "
```

Added a space between sentences.